packages/chrome-extension/src/react/ClerkProvider.tsx

```diff
@@ -7,21 +7,28 @@ import { createClerkClient } from '../internal/clerk';
 import type { StorageCache } from '../internal/utils/storage';
 
 type ChromeExtensionClerkProviderProps = ClerkReactProviderProps & {
+  /**
+   * @experimental
+   * @description Enables the listener to sync host cookies on changes.
+   */
+  ___experimentalSyncHostListener?: boolean;
```

Comment:
Let's respect the pattern and so 
```suggestion
  ___experimental_syncHostListener?: boolean;
```

Reply:
@panteliselef Ah, good catch!